docs/user-guide/src/bmo/provisioning.md

```diff
@@ -0,0 +1,127 @@
+spec:
+  online: true
+  bootMACAddress: 80:c1:6e:7a:e8:10
+  bmc:
+    address: ipmi://192.168.1.13
+    credentialsName: host-0-bmc
+  image:
+    checksum: http://192.168.0.150/SHA256SUMS
+    url: http://192.168.0.150/jammy-server-cloudimg-amd64.img
+```
+
+In most real cases, you will also want to provide
+
+1. first-boot configuration as described in [instance
+   customization](./instance_customization.md),
+2. [hints to choose the target root device](./root_device_hints.md),
+3. the format of the image you use.
+
+As a result, a more complete example will look like this:
+
+```yaml
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: host-0
+  namespace: my-cluster
+spec:
+  online: true
+  bootMACAddress: 80:c1:6e:7a:e8:10
+  bmc:
+    address: ipmi://192.168.1.13
+    credentialsName: host-0-bmc
+  image:
+    checksum: http://192.168.0.150/SHA256SUMS
+    url: http://192.168.0.150/jammy-server-cloudimg-amd64.img
+    format: raw
+  rootDeviceHints:
+    wwn: "0x55cd2e415652abcd"
+  userData:
+    name: host-0-userdata
+```
+
+When the provisioning state of the host becomes `provisioned`, your instance is
+ready to use. Note, however, that booting the operating system and applying the
+first boot scripts will take a few more minutes after that.
+
+### Note on images
+
+Two image formats are commonly used with Metal3: QEMU's
+[qcow2](https://en.wikipedia.org/wiki/Qcow) and raw disk images. Both formats
+have their upsides and downsides:
+
+- Qcow images are usually smaller and thus require less network bandwidth to
+  transfer, especially if you provision many machines with different images at
+  the same time.
+
+- Raw images can be streamed directly from the remote location to the target
+  block device without any conversion. However, they can be very large.
+
+When the format is omitted, Ironic will download the image into the local cache
+and inspect its format. If you want to use the streaming feature, you need to
+provide the `raw` format explicitly. If you want to forcibly cache the image
+(for example, because the remote image server is not accessible from the
+machine being provisioned), omit the format or use `qcow2` images.
+
+**HINT:** cloud-init is capable of [growing the last
+partition](https://cloudinit.readthedocs.io/en/latest/reference/modules.html#growpart)
+to occupy the remaining free space. Use this feature instead of creating very
+large raw images with a lot of empty space.
+
+**NOTE:** the special format value `live-iso` triggers a [live ISO
+provisioning](./live-iso.md) that works differently from a normal one.
+
+## Deprovisioning
+
+To remove an instance from the host and make it available for new deployments,
+remove the `image`, `userData`, `networkData`, `metaData` and `customDeploy`
+fields (if present). Depending on the host configuration, it will either start
+the [automated cleaning](./automated_cleaning.md) process or will become
+`available` right away.
+
+## Re-provisioning
```

Comment:
```suggestion
## Reprovisioning
```
Same with this spelling. Both are legit, but without dash is prevalent and also seems to be more common of the spellings. This occurs just few times, while reprovision occurs much more often.